testing/sdk_utils.py

```diff
@@ -48,6 +48,15 @@ def get_task_id_service_name(service_name):
     return service_name.lstrip("/").replace("/", ".")
 
 
+def get_scheduler_name(service_name):
```

Comment:
this will likely change when Nick merges #2616 but it's essentially the same thing as `get_scheduler_task_prefix`; for now this satisfies the ticket and fixing the broken test

Reply:
Would you mind linking the location in @nickbp's PR where you expect the changes to be made.

Reply:
https://github.com/mesosphere/dcos-commons/pull/2616/files#diff-e912dd60c30385c2608f3a5bed229478R276

Nick edited the get_scheduler_host method to create `get_scheduler_task_prefix`. It's just a different name but the logic is identical